source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: ember-leaflet
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber
+alias: 2019/12/17/countdown-to-the-new-year-2019-ember-leaflet
+responsive: true
+---
+
+The first of its kind "Countdown to The New Year: 31 Days of Ember Addons" series highlights popular addons in the Ember community. We hope you enjoy learning about fun tools for Ember developers together with us each day!
+
+## Day 17
+
+In the 17th increment of **Countdown to The New Year** let's take a look at an exciting Ember addon to add **maps** to our apps: **ember-leaflet**!
+
+Learn more about [ember-leaflet on Ember Observer](https://emberobserver.com/addons/ember-leaflet).
+
+### What It Does
+
+With ember-leaflet we can create useful maps for our web applications. This addon helps us to integrate the functionality of [Leaflet.js](https://leafletjs.com/) - an open-source JavaScript library for mobile-friendly, interactive maps - into our Ember app.
+
+### Why I Like It
+
+Leaflet.js is already a great library built with simplicity, performance and usability in mind. The addon ember-leaflet makes the developer experience even better for us: we can build complex, multi-layered street maps directly in our templates using the useful layer components provided by the addon API. This makes creating interactive map experiences much more straightforward and much more fun!
+
+Build your own map using ember-leaflet by [checking out the project description on Ember Observer!](https://emberobserver.com/addons/ember-exam)
```

Comment:
```suggestion
Build your own map using ember-leaflet by [checking out the project description on Ember Observer](https://emberobserver.com/addons/ember-exam)!
```

Reply:
The link points to ember-exam.

Reply:
Good catch @amyrlam and @tylerturdenpants - will update in a bit!